lms/djangoapps/instructor_task/models.py

```diff
@@ -198,16 +198,21 @@ def from_config(cls, config_name):
         config = getattr(settings, config_name, {})
         storage_type = config.get('STORAGE_TYPE', '').lower()
         if storage_type == 's3':
-            return DjangoStorageReportStore(
-                storage_class='openedx.core.storage.S3ReportStorage',
-                storage_kwargs={
-                    'bucket': config['BUCKET'],
-                    'location': config['ROOT_PATH'],
-                    'custom_domain': config.get("CUSTOM_DOMAIN", None),
-                    'querystring_expire': 300,
-                    'gzip': True,
-                },
-            )
+            if (
+                    hasattr(settings, 'S3_HOST') and settings.S3_HOST and
```

Comment:
Why do we need to test `hasattr` if it is added in current PR?

Reply:
Done